docs/src/modules/components/AppSearch.js

```diff
@@ -153,28 +154,33 @@ function NewStartScreen() {
   );
 }
 
-function DocSearchHit(props) {
-  const { children, hit } = props;
+const displayTagProductId = {
+  'material-ui': 'Material UI',
+  'joy-ui': 'Joy UI',
+  'base-ui': 'Base UI',
+  x: 'MUI X',
```

Comment:
Out of the scope of this PR, but I'm wondering if we should use `x-data-grid`,`x-data-charts`, `x-data-explore` in the future to split the big part of X docs that are independent one from the other

Reply:
I'm not sure, it feels like it would go too granular. I think that the badge is here to help developers quickly disambiguate if they are in the right place. The subheader already helps to orientes:

<img width="727" alt="Screenshot 2023-06-15 at 02 32 26" src="https://github.com/mui/material-ui/assets/3165635/2b6b6253-914f-4f1c-bcaa-18ca9e9547da">